client/src/elm/Pages/WellChildActivity/View.elm

```diff
@@ -1118,7 +1118,7 @@ viewVaccinationOverview language currentDate child vaccinationProgress db =
                         |> Maybe.map Tuple.second
 
                 nextDueText =
-                    Maybe.map formatDDMMYY nextDue
+                    Maybe.map formatDDMMyyyy nextDue
```

Comment:
Please add screenshots for changes at this file.
Since date becomes longer, we may need to adjust the styling.

Reply:
![Screen Shot 2021-12-01 at 15 15 42](https://user-images.githubusercontent.com/32199791/144241024-a95f382d-6f44-415b-a897-433c0366ba72.png)